packages/web-react/src/types/tooltip.ts

```diff
@@ -1,6 +1,16 @@
 import { Placement } from '@floating-ui/react';
 import { ChildrenProps, PlacementDictionaryType, StyleProps, ClickEvent } from './shared';
 
+export const TOOLTIP_TRIGGER = {
+  CLICK: 'click',
+  HOVER: 'hover',
+  MANUAL: 'manual',
+  OUTSIDE_PRESS: 'outside-press',
+  ESCAPE_KEY: 'escape-key',
+} as const;
+
+export type TooltipTriggerType = 'click' | 'hover' | 'manual';
```

Comment:
```suggestion
export type TooltipTriggerType = TOOLTIP_TRIGGER.CLICK | TOOLTIP_TRIGGER.HOVER  | TOOLTIP_TRIGGER.MANUAL ;
```
?

Reply:
i can't use it as you suggested because of errors:
```
Cannot find namespace 'TOOLTIP_TRIGGER'.ts(2503)
Exported type alias 'TooltipTriggerType' has or is using private name 'TOOLTIP_TRIGGER'.
```

but i can do:
```
export type TooltipTriggerType = Exclude<keyof typeof TOOLTIP_TRIGGER, 'OUTSIDE_PRESS' | 'ESCAPE_KEY'>;
```

what do you think?

Reply:
i think it's better as it was

Reply:
Ok, leave this as it is. :-)